app/jobs/worker/cdr_export_job.rb

```diff
@@ -0,0 +1,45 @@
+module Worker
+  class CdrExportJob < ActiveJob::Base
+    queue_as 'cdr_export'
+
+    attr_reader :cdr_export
+
+    def perform(cdr_export_id)
+      @cdr_export = find_cdr_export(cdr_export_id)
+
+      Cdr::Cdr.connection.execute("COPY (#{cdr_export.export_sql}) TO '#{file_path}' WITH CSV HEADER;")
```

Comment:
you need instance variable `@cdr_export` for file_path
why not just add method
```ruby
def file_path_for(cdr_export)
  "#{dir_path}/#{cdr_export.id}.csv"
end
```
an use it without unnecessary instance variables like this:
```ruby
cdr_export = find_cdr_export(cdr_export_id)
file_path = file_path_for(cdr_export)
Cdr::Cdr.connection.execute("COPY (#{cdr_export.export_sql}) TO '#{file_path}' WITH CSV HEADER;")
```